### DISCUSSION TITLE
小霸王 | /posts/small-overlord/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#606] [Announcements]
https://yuanfan.rbind.io/posts/small-overlord/

**Comment by wenbindu** [comment]
碎碎念，挺有趣

**Reply by earfanfan** [reply]
额，俺要一本正经地表达一下，谢谢你的夸赞。哈哈，我倒是没想到还有除我以外的人看得下去这些碎碎念。

**Comment by tianheg** [comment]
我已经有一段时间没有读你的文章了，阅读了这么多有趣的内容，我感到很开心。

**Reply by earfanfan** [reply]
你开心就好呀。另，我注意到你的头像改变了，记得以前是白色的圆里被黑色的直线分成三等分。

**Reply by tianheg** [reply]
现在的头像是Emacs（一个文档编辑器）的logo，改变头像是想提醒自己记得学习Emacs自定义相关的知识，不过好像没什么用 :)，有很长时间没接触Emacs相关的东西了

老头像是我自己设计的logo，最原始的设计想法已经忘了，有点后悔没记下来。只记得：

> 从中心到圆的边是人的一生，中心的圆和粗细变化的线条象征着人的变化。

而采用三，作为logo的数字自然是老子的“一生二，二生三，三生万物”。在圆中三是和谐的数字。

参考了 https://github.com/reuixiy 的头像

**Reply by earfanfan** [reply]
哇，原来你之前的头像是花了这么多心思设计的啊，难怪令我印象深刻记了这么久，有种大美至简的味道。现在的这个要不是你解释了一番，在我看来很像是希腊字母衣服洗桶，带点学术的味道。

**Reply by tianheg** [reply]
> 现在的这个要不是你解释了一番，在我看来很像是希腊字母衣服洗桶，带点学术的味道。

你这句，我第一次读没读懂。又读一次，哈哈，太有趣了😃
